CHANGES_NEXT_RELEASE

```diff
@@ -6,7 +6,7 @@
 - Add: reset for notification counters in threadpool mode
 - Fix: avoid -g compiler flag in release build
 - Add: proper alarm management, including activation/deactivation in logs (Issue #1582)
-- Fix: improved JSON representation for APIv2 (Issue #1259)
+- Fix: entity/attribute operations to align them with the last JSON representation format defined for NGSIv2 (Issue #1259)
```

Comment:
I have changed the old message for something more descriptive. Improved versions are welcome :)